opennms-container/horizon/container-fs/confd/conf.d/newts.properties.toml

```diff
@@ -3,9 +3,5 @@ src = "newts.properties.tpl"
 dest = "/opt/opennms/etc/opennms.properties.d/_confd.newts.properties"
 keys = [
     "replication/factor",
-    "opennms/cassandra/hostnames",
-    "opennms/cassandra/keyspace",
-    "opennms/cassandra/port",
-    "opennms/cassandra/username",
-    "opennms/cassandra/password"
+    "opennms/cassandra",
```

Comment:
Does the trailing `,` make issues? I think with TOML it doesn't matter. We just should make sure :)

Reply:
I think it does not make issues, since we have it here too: https://github.com/OpenNMS/opennms/pull/3228/files#diff-ef77a3aec76fce5e3fc7ec2b6b309d37be42babd508761ac8355561b2dc70a17R5
But we also have it here without the trailing `,`.

Reply:
```suggestion
    "opennms/cassandra"
```